client/sections-preload.js

```diff
@@ -1,7 +1,7 @@
 /**
  * sections-preload
  *
- * This is a simple eventbus that sections.js listens to to know when to preload sections.
+ * This is a simple eventbus that sections.js listens to know when to preload sections.
```

Comment:
I think this just needs a comma between the two to's, or to be reworded, as even with this change it doesn't read well.